### PR TITLE
Document Celery over systemd instead of Celery over uWSGI

### PR DESCRIPTION
**Pull request by noirbizarre**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -91,17 +91,19 @@ All you need to remember is that uData requires at least 3 services to run:
 - a worker service using [celery][]
 - a beat/cron service using [celery][] too
 
-We give you an example using [nginx][] + [uWSGI][] to run these 3 parts,
-expose the frontend on the `data.example.com` domain
-(and having the middlewares running on the same host)
+We give you an example using [nginx][] + [uWSGI][] to run the frontend
+on the `data.example.com` domain
+systemd both the worker and the beat services,
```
Comment (abulte) [inline docs/installation.md]: Not a sentence?

File: docs/installation.md [hunk 2]
```diff
@@ -154,67 +155,65 @@ vacuum = true
...
-/tmp/celery-worker.pid %(home)/bin/celery -A udata.worker -Q high,default worker
-/tmp/celery-worker.pid %(home)/bin/celery -A udata.worker -Q high,default,low worker
+```shell
+$ ln -s /etc/uwsgi/apps-{available,enabled}/udata-front.ini
 ```
 
+You can now create the systemd unit file for Celery:
 
-**`/etc/uwsgi/apps-available/udata-beat.ini`**
+**`/etc/systemd/system/celery.service`**
```
Comment (abulte) [inline docs/installation.md]: `udata-celery.service`? To match our config.

File: docs/installation.md [hunk 3]
```diff
@@ -154,67 +155,65 @@ vacuum = true
...
-exec-as-user-atexit = kill -TERM `cat /tmp/celery-beat.pid`
+[Unit]
+Description=udata celery services
+After=network.target
+
+[Service]
+Type=forking
+User=udata
+WorkingDirectory=/srv/udata
+LogsDirectory=udata-celery
```
Comment (abulte) [inline docs/installation.md]: In prod right now `[/etc/systemd/system/udata-celery.service:10] Unknown lvalue 'LogsDirectory' in section 'Service'`

File: docs/installation.md (hunk 2 above)
Comment (noirbizarre) [reply docs/installation.md]: I tried to simply it a little bit to make it easier to read

File: docs/installation.md (hunk 3 above)
Comment (noirbizarre) [reply docs/installation.md]: Ignored because of Debian old Systemd version but required on some newer versions.

File: docs/installation.md (hunk 1 above)
Comment (noirbizarre) [reply docs/installation.md]: I've rewritten the whole paragraph

File: docs/installation.md [hunk 4]
```diff
@@ -91,17 +91,20 @@ All you need to remember is that uData requires at least 3 services to run:
 - a worker service using [celery][]
 - a beat/cron service using [celery][] too
 
-We give you an example using [nginx][] + [uWSGI][] to run these 3 parts,
-expose the frontend on the `data.example.com` domain
-(and having the middlewares running on the same host)
+We give you an example for a `udata` user serving a `data.example.com` domain from `/srv/udata` on a single server with:
+
+- [nginx][] + [uWSGI][] to run the frontend
+- systemd handling both the worker and the beat services,
```
Comment (abulte) [inline docs/installation.md]: `,` above too or remove this one.

**Pull request merged**